### DISCUSSION TITLE
Map writer make a cut down map

### DISCUSSION THREAD

**Discussion by abbdef** [mapsforge/mapsforge#1396] [Q&A] [answered]
I need to make a cut down map that shows only road network to get a GPS navigation app functioning. I'm wondering if there were any examples of how people use mapwriter commands to make such maps. Which tags etc.

**Comment by devemux86** [comment, marked as answer]
If it is just about the map layers visibility, it is easier to customize the [render theme](https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md).

If it is about the content of the map file, it is handled by the [tag-mapping](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-map-writer/src/main/config/tag-mapping.xml), that can be passed externally in [map-writer](https://github.com/mapsforge/mapsforge/blob/master/docs/Getting-Started-Map-Writer.md).

**Reply by abbdef** [reply]
The main thing is to get the .map file size to about 300mb. That's the only reason. For some reason the country's  .map file is about 1.8G .
